roxctl/common/zipdownload/download_zip.go

```diff
@@ -118,16 +115,19 @@ func GetZip(opts GetZipOptions) error {
 	zipFileName, err := download.ParseFilenameFromHeader(resp.Header)
 	if err != nil {
 		zipFileName = fmt.Sprintf("%s.zip", opts.BundleType)
-		printf("Warning: could not obtain output file name from HTTP response: %v.", err)
-		printf("Defaulting to filename %q", zipFileName)
+		common.CLIEnvironmentSingleton().Logger().WarnfLn(
+			"Warning: could not obtain output file name from HTTP response: %v", err)
```

Comment:
```suggestion
			"could not obtain output file name from HTTP response: %v", err)
```

Reply:
Just an idea for a separated task if we have debug mode it will be great to use `%+v` to print stacktrace